ddht/tools/web3.py

```diff
@@ -69,17 +74,77 @@ def from_rpc_response(cls, response: TableInfoResponse) -> "TableInfo":
         )
 
 
+class PongPayload(NamedTuple):
+    enr_seq: int
+    packet_ip: AnyIPAddress
+    packet_port: int
+
+    @classmethod
+    def from_rpc_response(cls, response: PongResponse) -> "PongPayload":
+        return cls(
+            enr_seq=response["enr_seq"],
+            packet_ip=ipaddress.ip_address(response["packet_ip"]),
+            packet_port=response["packet_port"],
+        )
+
+
 class RPC:
     nodeInfo = RPCEndpoint("discv5_nodeInfo")
     routingTableInfo = RPCEndpoint("discv5_routingTableInfo")
 
+    ping = RPCEndpoint("discv5_ping")
+
+
+def ping_munger(
+    module: Any, identifier: Union[ENRAPI, str, bytes, NodeID, HexStr]
+) -> List[str]:
```

Comment:
low priority but it took me a bit to figure out why this was returning a list. A short description, or a link to https://github.com/ethereum/web3.py/blob/master/web3/method.py#L77 would have saved me  a couple minutes, and might save the time of a hypothetical future reader.

Reply:
cc @kclowes or @marcgarreau 

This is an area of the web3.py internals that *could* be documented at some point assuming we want to encourage people to extend the library like this.  I struggled a bit to figure out how to write a `Method(...)` properly and only got there by futzing and copy/pasta from the web3.py source code.

Reply:
Yeah, that's a good idea! We can definitely document it. I think it's generic enough that we could allow people to extend with a minimal amount of effort. Added issue ethereum/web3.py#1755